tests/load_data/tracking_data/test_add_player_data_to_dict.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+import numpy as np
+
+from databallpy.load_data.tracking_data._add_player_data_to_dict import _add_player_data_to_dict
+
+
+
+class TestTracab(unittest.TestCase):
```

Comment:
`class TestAddPlayerDataToDict`, maybe needs to be changed if you change the file name. The format for test names is f"Test{filename}" with camel case.  The functions in the class are f"test_{function name in file you are testing}" with snake case